package.json

```diff
@@ -5,7 +5,7 @@
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build --prod",
-    "test": "ng test --no-progress --no-watch --code-coverage",
+    "test": "ng test --no-progress --no-watch --code-coverage --project ngx-valdemort",
```

Comment:
FYI, the --project is not needed. You can pass the project directly, without this prefix